frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -29,8 +30,10 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
   const submit = (event: React.FormEvent<EventTarget>) => {
     event.preventDefault();
     setProgress(true);
-    const presentCapacity = _.get(ocsConfig, 'spec.storageDeviceSets[0].count');
-    const newValue = parseInt(presentCapacity, 10) + parseInt(requestSizeValue, 10) * 3;
+    const presentCount = _.get(ocsConfig, 'spec.storageDeviceSets[0].count');
+    const negativeValue = Number(requestSizeValue) < 0 ? -1 : 1;
+    const newValue =
+      (parseInt(presentCount, 10) + Math.abs(Number(requestSizeValue)) * 3) * negativeValue;
```

Comment:
Any reason for using different functions `parseInt` and `Number` for converting to number in a single line. I feel we should stick with a single method only if it fulfills the requirement.

Reply:
ack, will use `Number` in both places